lib/reporter.js

```diff
@@ -225,6 +226,13 @@ class CucumberReporter {
         })
     }
 
+    getTitle (featureOrScenario) {
+        const name = featureOrScenario.getName()
+        const tags = featureOrScenario.getTags ? featureOrScenario.getTags() : []
```

Comment:
Fair enough. Updates in last commit. 

https://github.com/cucumber/cucumber-js/blob/1.x/lib/cucumber/ast/feature.js#L61
https://github.com/cucumber/cucumber-js/blob/1.x/lib/cucumber/ast/scenario.js#L39

^ implementation for getTags in cucumber 1.x